frontend/app/templates/components/transportation/study-area-map/census-tract-data.hbs

```diff
@@ -0,0 +1,5 @@
+{{#throttle-property property=geoid as |geoid|}}
```

Comment:
I like that it only throttles data loading, and not the popup itself. Also that this is expressed by the template!